config/cardinal_pooling.csv

```diff
@@ -0,0 +1,97 @@
+number, pool_1, pool_2, pool_3, pool_4, pool_5, pool_6, pool_7, pool_8
+96,12,12,12,12,12,12,12,12
+95,12,12,12,12,12,12,12,11
+94,12,12,12,12,12,12,11,11
+93,12,12,12,12,12,11,11,11
+92,12,12,12,12,11,11,11,11
+91,12,12,12,11,11,11,11,11
+90,12,12,11,11,11,11,11,11
+89,12,11,11,11,11,11,11,11
+88,11,11,11,11,11,11,11,11
+87,13,13,13,12,12,12,12,
+86,13,13,12,12,12,12,12,
+85,13,12,12,12,12,12,12,
+84,12,12,12,12,12,12,12,
+83,12,12,12,12,12,12,11,
+82,12,12,12,12,12,11,11,
+81,12,12,12,12,11,11,11,
+80,12,12,12,11,11,11,11,
+79,12,12,11,11,11,11,11,
+78,12,11,11,11,11,11,11,
+77,11,11,11,11,11,11,11,
+76,13,13,13,13,12,12,,
+75,13,13,13,12,12,12,,
+74,13,13,12,12,12,12,,
+73,13,12,12,12,12,12,,
+72,12,12,12,12,12,12,,
+71,12,12,12,12,12,11,,
+70,12,12,12,12,11,11,,
+69,12,12,12,11,11,11,,
+68,12,12,11,11,11,11,,
+67,12,11,11,11,11,11,,
+66,11,11,11,11,11,11,,
+65,13,13,13,13,13,,,
+64,13,13,13,13,12,,,
+63,13,13,13,12,12,,,
+62,13,13,12,12,12,,,
+61,13,12,12,12,12,,,
+60,12,12,12,12,12,,,
+59,12,12,12,12,11,,,
+58,12,12,12,11,11,,,
+57,12,12,11,11,11,,,
+56,12,11,11,11,11,,,
+55,11,11,11,11,11,,,
+54,11,11,11,11,10,,,
+53,11,11,11,10,10,,,
+52,13,13,13,13,,,,
+51,13,13,13,12,,,,
+50,13,13,12,12,,,,
+49,13,12,12,12,,,,
+48,12,12,12,12,,,,
+47,12,12,12,11,,,,
+46,12,12,11,11,,,,
+45,12,11,11,11,,,,
+44,11,11,11,11,,,,
+43,11,11,11,10,,,,
+42,11,11,10,10,,,,
+41,11,10,10,10,,,,
+40,10,10,10,10,,,,
+39,13,13,13,,,,,
+38,13,13,12,,,,,
+37,13,12,12,,,,,
+36,12,12,12,,,,,
+35,12,12,11,,,,,
+34,12,11,11,,,,,
+33,12,11,10,,,,,
+32,12,10,10,,,,,
+31,11,10,10,,,,,
+30,10,10,10,,,,,
+29,10,10,9,,,,,
+28,10,9,9,,,,,
+27,9,9,9,,,,,
+26,13,13,,,,,,
+25,13,12,,,,,,
+24,12,12,,,,,,
+23,12,11,,,,,,
+22,11,11,,,,,,
+21,10,11,,,,,,

```

Comment:
Is this one right?  It's the only one I can see where the numbers are not in equal-or-descending order.